utilities/inference_utils.py

```
@@ -574,7 +574,8 @@ def create_isvc(
     autoscaler_mode: str | None = None,
     stop_resume: bool = False,
     multi_node_worker_spec: dict[str, Any] | None = None,
-    timeout: int = Timeout.TIMEOUT_15MIN,
+    #timeout: int = Timeout.TIMEOUT_15MIN,
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)